src/alpaca_eval/leaderboards/data_AlpacaEval_2/weighted_alpaca_eval_gpt4_turbo_leaderboard.csv

```diff
@@ -1,212 +1,214 @@
 ,win_rate,standard_error,n_wins,n_wins_base,n_draws,n_total,discrete_win_rate,mode,avg_length,length_controlled_winrate,lc_standard_error
```

Comment:
somehow all the leaderboard seems to have changed because the n_total went from int to float. Can you check why / solve it/ and rebase on main?

Reply:
I checked the history of this file and it seems that the float issue is due to the new result from #405 which contains an incorrect float number 805.0, after fixing this number everything seems fine. Have updated the correct leaderboard now.